static/app/utils/integrationUtil.tsx

```diff
@@ -11,6 +11,7 @@ import {
   IconJira,
   IconVsts,
 } from 'sentry/icons';
+import {IconCodecov} from 'sentry/icons/iconCodecov';
```

Comment:
you'll have to merge this with the other imports
```suggestion
import {IconCodecov} from 'sentry/icons';
```